docs/developer/partition_api.rst

```diff
@@ -1,7 +1,7 @@
 Partition API in Modin
 ======================
 
-When you are working with a Modin Dataframe, you can unwrap its remote partitions
+When you are working with a Modin DataFrame, you can unwrap its remote partitions
```

Comment:
I prefer to revert this change. DataFrame is camel case  because of "data frame". I prefer "dataframe".

Reply:
Okay, I do not mind. Probably, we will need to fix `DataFrame` -> `dataframe` in many places with one commit after we complete documentation changes :)

Reply:
Created https://github.com/modin-project/modin/issues/3070 to track.